content/docs/get-started/aws/modify-program.md

```diff
@@ -18,44 +18,51 @@ Now that your S3 bucket is provisioned, let's add an object to it. First, create
 $ mkdir site
 ```
 
-Next create an `index.html` file you will upload to your bucket.
+Next, create a new `index.html` file with some content you will upload to your bucket.
 
 {{< chooser os "macos,linux,windows" / >}}
 
 {{% choosable os macos %}}
 
 ```bash
-$ touch site/index.html
+$ cat <<EOT > site/index.html
+<html>
+    <body>
+        <h1>Hello, Pulumi!</h1>
+    </body>
+</html>
+EOT
 ```
 
 {{% /choosable %}}
 
 {{% choosable os linux %}}
 
 ```bash
-$ touch site/index.html
+$ cat <<EOT > site/index.html
+<html>
+    <body>
+        <h1>Hello, Pulumi!</h1>
+    </body>
+</html>
+EOT
 ```
 
 {{% /choosable %}}
 
 {{% choosable os windows %}}
 
-```bash
-$ type nul > site/index.html
+```bat
+> site/index.html (
+    @echo.^<html^>
+    @echo.  ^<body^>
+    @echo.      ^<h1^>Hello, Pulumi!^</h1^>
+    @echo.  ^</body^>
+    @echo.^</html^>)
```

Comment:
Not a huge deal - but these may make the prose slightly harder to use in IDE-like environments (vs. directly in a shell) - especially this Windows one which can't be pasted into an IDE/editor.  I wonder if simplifying this in the other direction would be better "Create a file named `site/index.html` and add the following text".

Reply:
It feels like we should be okay here because the preceding steps in the tutorial assume people are in a shell to use the Pulumi CLI, make directories, etc.

Btw, I have updated the text below this section to make sure folks know to transition to an IDE or text editor.